CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   `Mosaic` component is now correctly displayed on IE11.
 -   Fix missing grey color (in material theme) in the generated SCSS color palette
 
+### Changed
+-   _[BREAKING]_ Changed `thumbnailProps` prop type from `LinkPreview` to avoid typescript throwing an error
+if `thumbnailProps.image` was not specified even though it is already set by the `image` prop.
+
```

Comment:
Must be in the "unreleased" section of the changelog

Reply:
Oops, I though it was.
Fixed

Reply:
Yeah it often occurs on rebase since the version heading is added just under the "unrelease" heading